src/jit/morph.cpp

```diff
@@ -6893,6 +6893,11 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
     }
 #endif
 
+    assert(!callee->AreArgsComplete());
```

Comment:
Does the function header comment need any updates?

Reply:
Updated.